src/components/image/index.tsx

```diff
@@ -218,10 +218,11 @@ class Image extends PureComponent<Props, State> {
           this.isGif() && styles.gifImage,
           aspectRatio && {aspectRatio},
           !useImageInsideContainer && margins,
+          useImageInsideContainer && {resizeMode: 'contain'},
```

Comment:
When setting a static style that doesn't depends on a prop you can move it to the file's StyleSheet

Reply:
`
          useImageInsideContainer && {resizeMode: 'contain'},
          style,
          useImageInsideContainer && styles.shrink
`

In this case I'll add the resizeMode to styles.shrink since they depend on the same boolean right? 

like this:
`
  shrink: {
    flexShrink: 1,
    resizeMode: 'contain'
  }
`